packages/datagateway-dataview/src/views/card/cardView.component.tsx

```diff
@@ -789,17 +787,28 @@ const CardView = (props: CardViewProps): React.ReactElement => {
                   );
                 })}
               </List>
-            </Grid>
+            ) : (
+              <Grid xs={12} md={8}>
+                <Paper className={classes.noResultsPaper}>
+                  <Typography align="center" variant="h6" component="h6">
+                    {t('loading.filter_message')}
```

Comment:
Yeah I don't see any reason why we shouldn't the message there too, but might require some changes to the layout because of how we display the table compared to CV. So as you say probably best having a seperate issue, I've made a comment on the original issue to make this change (and any other similar improvements) to table view as well.